react/features/base/dialog/components/DialogContent.native.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+
+import React, { Component } from 'react';
+import { Text, View } from 'react-native';
+
+import { dialog as styles } from './styles';
+
+type Props = {
+
+    /**
+     * Children of the component.
+     */
+    children: string | React$Node
+};
+
+/**
+ * Generic dialog content container to provide the same styling for all custom
+ * dialogs.
+ */
+export default class DialogContent extends Component<Props> {
+    /**
+     * Implements {@code Component#render}.
+     *
+     * @inheritdoc
+     */
+    render() {
+        const { children } = this.props;
+
+        const childrenComponent = typeof children === 'string'
+            ? <Text>{ children }</Text>
+            : children;
+
+        return (
+            <View style = { styles.dialogContainer }>
+                { childrenComponent }
+            </View>
+        );
+    }
```

Comment:
Curious, what would you think the web implementation would look like if it were to implement DialogContent? The purpose of its existence for now is to auto-wrap text in <Text>?

Reply:
The main reason for it on mobile was to provide a consistent look (same padding, margin, layout). Basically intended to replace ```styles.messageContainer``` in the recording related dialogs too, that is used over and over again (scheduled for a follow up PR). I guess web could use something similar, if need be